src/app.js

```diff
@@ -50,15 +53,22 @@ app.post('/create-game', async (req, res) => {
 
 app.get('/my-started-game', async (req, res) => {
   try {
-    const { userId } = req.query
-    const startedGames = await dynamo.getStartedGames(userId)
+    const { playerId } = req.query
+    if (!playerId) {
+      return res.status(400).send({ message: 'Field playerId is required!' })
+    }
+    const startedGames = await dynamo.getStartedGames(playerId)
     const gamesId = startedGames.map(({ id: gameId }) => {
       return { gameId }
     })
-    res.send(gamesId)
+    if (gamesId.length !== 0) {
+      res.send(gamesId)
+    } else {
```

Comment:
@SlavenGrowy In Java Calc Academy we used `{}` everywhere, so that is perhaps a practice we can continue as this leaves out the possibility to add confusion by adding statements before/after the oneliner and do a complete mess :)

But as you mentioned this is subjective and a matter of code style that the team should adopt and follow.